src/viur/core/utils.py

```diff
@@ -273,6 +273,29 @@ def normalizeKey(key: Union[None, 'db.KeyClass']) -> Union[None, 'db.KeyClass']:
     return db.Key(key.kind, key.id_or_name, parent=parent)
 
 
+def prefixes(name: str, prefix: str, delimiter: str = ".") -> bool:
```

Comment:
I don't find the name very meaningful. `prefixes` sounds like prefixes are being generated here -- even though the docstring explains that, of course.

technically it should be `is_equal_or_prefix`. But maybe `is_prefix` is already enough. 

```suggestion
def is_prefix(name: str, prefix: str, delimiter: str = ".") -> bool:
```